info/v1/container.go

```diff
@@ -848,6 +848,25 @@ type PerfStat struct {
 	Cpu int `json:"cpu"`
 }
 
+type MemoryBandwidthMonitoringStats struct {
```

Comment:
Maybe MemoryBandwidthStats?  Or is this metrics about the monitoring of memory bandwidth?

Reply:
It's related to MBM ( Memory Bandwidth Monitoring ). See: https://01.org/cache-monitoring-technology

Reply:
Based on that, we should call this MemoryBandwithStats.  I realize this ruins the appreciation, but in this context, "monitoring" is unnecessary